### ISSUE TITLE
 Improve form validation and submission process to prevent duplicate submissions

### ISSUE THREAD

**Issue by samsara24** [lxmghct/user-role-admin#1]
**Issue:**
In the `updatePassword` method located in `manage-frontend/src/components/PasswordUpdateDialog/index.vue`, the form validation is followed by an asynchronous operation that changes the user's password by calling the `changePassword` function. However, during this process, there's a potential issue where the user could quickly click the "Confirm" button multiple times, leading to duplicate submissions of the password change request. This could result in unnecessary API calls or unintended behavior.

**Proposed Solution:**

To prevent duplicate submissions, it is recommended to:

1. **Disable the "Confirm" Button**: Once the user clicks the "Confirm" button and the form validation begins, disable the button to prevent further clicks until the asynchronous operation completes.

2. **Re-enable the Button**: After the asynchronous operation (i.e., the password change request) is completed, re-enable the button so that the user can interact with it again if necessary.

**Example Implementation:**

```javascript
methods: {
  updatePassword() {
    this.$refs['passwordUpdateForm'].validate((valid) => {
      if (valid) {
        this.isSubmitting = true; // Disable the button
        changePassword(this.passwordUpdateForm.oldPassword, this.passwordUpdateForm.newPassword)
          .then(() => {
            this.$message.success('密码修改成功');
            this.cancelUpdatePassword();
          })
          .finally(() => {
            this.isSubmitting = false; // Re-enable the button
          });
      }
    });
  }
}

**Comment by lxmghct** [comment]
Thank you for reporting this issue. The bug was fixed in commit 66389a9. Please review the changes and let us know if the problem persists. 
Issue closed.

**Comment by samsara24** [comment]
Great solution! It effectively addresses the issue
